### PR TITLE
Update README link to point at new path

### PR DESCRIPTION
**Pull request by s-rubenstein**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,13 +40,13 @@ Requests related to the funtionality or usability of the UI can be submitted as
     yarn start
     ```
 4. Testing:
-
```
Comment (s-rubenstein) [inline README.md]: I don't know why it made these changes, but it seems to just be white space.

Comment (zarsky-broad) [reply README.md]: Hmm. Can you revert these lines to just avoid extra diff?

Comment (s-rubenstein) [reply README.md]: Done

**Review by panentheos — APPROVED** [review]
Looks good, thanks for keeping this up to date.

**Pull request merged**